### ISSUE TITLE
ArgumentNullException: Value cannot be null. Parameter name: message

### ISSUE THREAD

**Issue by ArmanHayots** [alexanderkozlenko/aspnetcore-json-rpc#4]
### Summary

While investigating abilities to send and receive objects using JSON-RPC, I found this strange error and I can't guarantee that it isn't a bug. It not says that syntax is wrong or something like this, it just crashes.

### Artifacts

```
Raw exception details 
System.ArgumentNullException: Value cannot be null.
Parameter name: message
   at Anemonis.JsonRpc.JsonRpcError..ctor(Int64 code, String message)
   at Anemonis.AspNetCore.JsonRpc.JsonRpcMiddleware`1.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
```

### Code Example

```
public class TestService : IJsonRpcService
{
[JsonRpcMethod("test3", "ob")]
        public Task<bool> AnotherTest(string ob)
        {
            return Task.FromResult(ob != null); 
        }
}
```
and a query body 
```
//be noticed on wrong params syntax - [] instead of {}
{"jsonrpc":"2.0","id":"1","method":"test3","params":["ob":"test"]} 
```

**Comment by alexanderkozlenko** [comment]
I've published a new version of the package - 1.14.1, could you please check if it solves the issue for you?

**Comment by ArmanHayots** [comment]
Yes, it works correct now. Thank you.
